Add google authd-broker documentation (#724)
ubuntu/authd: docs/reference/troubleshooting.md

journalctl -u snap.authd-msentraid.authd-msentraid.service
```

> For the Google IAM broker entries, run:
> 
> ```shell
> journalctl -u snap.authd-google.authd-google.service
> ```
> 

Reviewer: ```suggestion
For the broker entries, run the following command, replacing `<broker-name>` with the appropriate broker, such as `google` or `msentraid`:

```shell
journalctl -u snap.authd-<broker-name>.authd-<broker-name>.service
```

```

Placeholders are used elsewhere in this reference, so we can probably do the same here instead of having two unique entries for each of the two brokers.

As this will be the first use of placeholders in the page, it's helpful to be explicit about the placeholders needing to be substituted.

Then you need to restart the service with `sudo systemctl restart gdm`.

> #### authd-<broker> service

Reviewer: ```suggestion
#### authd broker service
```

Avoid using angle brackets in headers: it doesn't render.

It also doesn't make sense to have placeholder syntax in text that the user will not be copying and running.

To increase the verbosity of the broker service, edit the service file:

```shell
sudo systemctl edit snap.authd-<broker_name>.authd-<broker_name>.service
```

Add the following lines to the override file and make sure to add `-vv` to the exec command:

```
[Service]
ExecStart=
ExecStart=/usr/bin/snap run authd-<broker_name> -vv
```

You will then need to restart the service with `snap restart authd-<broker_name>`.

## Switch the snap to the edge channel

Maybe your issue is already fixed! You should try switching to the edge channel of the broker snap. You can easily do that with:

```shell
snap switch authd-<broker_name> --edge
snap refresh authd-<broker_name>
```

> Keep in mind that this version is not tested and may be incompatible with current released version of authd. You should switch back to stable after trying the edge channel:

Reviewer: ```suggestion
Keep in mind that this version is not tested and may be incompatible with the current released version of authd. You should switch back to stable after trying the edge channel:
```
